cmd/configmap-server/main.go

```diff
@@ -98,7 +98,7 @@ func runCmdFunc(cmd *cobra.Command, args []string) error {
 		logger.Fatalf("error getting configmap: %s", err)
 	}
 
-	sqlLoader, err := sqlite.NewSQLLiteLoader(dbName)
+	sqlLoader, err := sqlite.NewSQLLiteLoader(dbName, "")
```

Comment:
We use some different patterns in other places that I think could clean this up. 

Either a builder with good defaults

```go
// uses the compiled migrations
sqlite.NewSQLLiteLoader(dbName)

// for tests
sqlite.NewSQLLiteLoader(dbName).WithMigrator(*testMigrator)
```
or the variadic option pattern
```go
sqlite.NewSQLLiteLoader(options.WithDBName(dbName), options.WithMigrator(default))
```

Reply:
Yep, this is a good point. I went with the trivial implementation instead of using a builder because the constructor currently has actual parsing logic that requires the migrator to exist since there is currently no distinction between calling `NewSQLLiteLoader` and actually initializing the database -- I was attempting to retain that step as an atomic function call instead of splitting it into several steps.

But using the option pattern is a really good idea, I'll try to make this more digestible for the default case that way.